### ISSUE TITLE
Vote shop and items in the game are different. 

### ISSUE THREAD

**Issue by NewbsUp** [dalaranwow/dalaran-wow#5356]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Go to vote shop and find item Grand Staff of Jordan. Purchase said item and compair the stats in the game. In the Shop this item has stats of int, sta, and Crit. In the game the item has Stam, Hit rating, Resilience, and spell power.. See the issue. So I purchased the item in to be a pve item, however the item is pvp.  
2. Step 2: Write about it on this page and get told from admin to open a ticket in the game. 
3.  Step 3: Open ticket in game and wait three weeks putting on a lot of play time and get no response. 
Being one of the players that has donated because I felt at the time the people who made this server Should have some compensation for the work they have done. I feel like my donations should have something to show and sense I have taken the proper actions asked of me I still do not understand why this is an issue and has not been fixed. Please help. 

Is it at all posible to get a refund of points or just have the item swapped out with the Dignified headmasters( The item that really holds those stats)?
![headmasters staff](https://user-images.githubusercontent.com/37151958/37629378-1d6d03ea-2ba4-11e8-84df-543dc1fa962c.jpg)
![in game staff](https://user-images.githubusercontent.com/37151958/37629385-22ffdc74-2ba4-11e8-8b62-e07690bf789b.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by NewbsUp** [comment]
![staff of jordan](https://user-images.githubusercontent.com/37151958/37629405-36d0f13e-2ba4-11e8-8dc3-873c6706b812.jpg)

**Comment by Rushor** [comment]
-- GM ONLY --
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
